src/types/index.ts

```diff
@@ -6,6 +6,8 @@ export enum BscChainId {
   'TESTNET' = 97,
 }
 
+export type Environments = 'testnet' | 'preview' | 'mainnet';
```

Comment:
Don't we have `staging` and `production` as well? I'm referring to Corey's message in the frontend channel.
Nit: it should be singularized: `Environment`

Reply:
In my opinion it's just confusing to add more than the 3 we use on the frontend right now. I'd update as needed.

Renamed to `Environment`